docker/public/custom_index.php

```diff
@@ -0,0 +1,12 @@
+<?php
+require_once('WOVN.php/src/wovn_interceptor.php');
+
+$content = file_get_contents('php://input');
+
+$response_json = json_decode($content, true);
+$response_options = $response_json['response'];
+foreach ($response_options['headers'] as $key => $value) {
+    header("{$key}: {$value}");
+}
+header("Content-Type: {$response_options['content_type']}");
```

Comment:
wovnrb version is `content-type`. We should keep the parameter names exactly the same

Reply:
Headers are case insensitive by definition, but it's Ok to change this.

Reply:
i'm talking about the body parameter sent by the browser

Reply:
Oops!